x-pack/plugins/monitoring/server/static_globals.ts

```diff
@@ -5,20 +5,50 @@
  * 2.0.
  */
 
-import { CoreSetup, ILegacyCustomClusterClient, Logger } from 'kibana/server';
+import {
+  CoreSetup,
+  ElasticsearchClient,
+  Logger,
+  SharedGlobalConfig,
+  PluginInitializerContext,
+} from 'kibana/server';
 import url from 'url';
-import { CloudSetup } from '../../cloud/server';
+import { estypes } from '@elastic/elasticsearch';
 import { MonitoringConfig } from './config';
-
+import { PluginsSetup } from './types';
+import { mbSafeQuery } from './lib/mb_safe_query';
 type GetLogger = (...scopes: string[]) => Logger;
 
+interface InitSetupOptions {
+  initializerContext: PluginInitializerContext;
+  coreSetup: CoreSetup;
+  config: MonitoringConfig;
+  getLogger: GetLogger;
+  log: Logger;
+  legacyConfig: SharedGlobalConfig;
+  setupPlugins: PluginsSetup;
+}
+
+export type EndpointTypes =
+  | 'search'
+  | 'msearch'
+  | 'transport.request'
+  | 'cluster.putSettings'
+  | 'cluster.getSettigns'
```

Comment:
typo here?

Reply:
Good catch, I put a typo canary in every PR just to keep everyone on their toes :P
(just kidding)